app/src/test/java/org/oppia/android/app/player/state/StateFragmentLocalTest.kt

```diff
@@ -352,6 +355,32 @@ class StateFragmentLocalTest {
     }
   }
 
+  @Test
+  @Config(qualifiers = "+port")
+  fun testStateFragment_portrait_submitCorrectAnswer_correctIsAnnounced() {
```

Comment:
Can you also add tests for the question player in QuestionActivityTest? You updated a test question but then didn't add a test for it.

Reply:
Done.